### PR TITLE
Use sAMAccountName

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: tests/foreman/cli/test_ldapauthsource.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def test_positive_refresh_usergroup_with_ad(self, member_group, ad_data):
                 'attr-firstname': LDAP_ATTR['firstname'],
                 'attr-lastname': LDAP_ATTR['surname'],
                 'attr-mail': LDAP_ATTR['mail'],
-                'account': ad_data['ldap_user_name'],
+                'account': fr"{ad_data['workgroup']}\{ad_data['ldap_user_name']}",
```
Comment (adarshdubey-star) [inline tests/foreman/cli/test_ldapauthsource.py:130]: This made me learn something about formatted and raw strings together @lhellebr :)

**Review by adarshdubey-star — APPROVED** [review]
Ack

**Comment by adarshdubey-star** [comment]
Let's have a PRT run and we can merge this!

File: tests/foreman/cli/test_ldapauthsource.py (hunk 1 above)
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ldapauthsource.py:130]: Nitpick comment about flipping the strings :)
```suggestion
                'account': fr'{ad_data["workgroup"]}\{ad_data["ldap_user_name"]}',
```

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ldapauthsource.py::test_positive_refresh_usergroup_with_ad

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ldapauthsource.py::test_positive_refresh_usergroup_with_ad

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ldapauthsource.py::TestADAuthSource::test_positive_refresh_usergroup_with_ad

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ldapauthsource.py::TestADAuthSource::test_positive_refresh_usergroup_with_ad

**Review by sambible — APPROVED** [review]

**Pull request merged**